libraries/AP_RPM/AP_RPM.h

```diff
@@ -38,11 +39,10 @@ class AP_RPM
     // RPM driver types
     enum RPM_Type {
         RPM_TYPE_NONE    = 0,
-        RPM_TYPE_PWM     = 1,
-        RPM_TYPE_PIN     = 2,
-        RPM_TYPE_EFI     = 3,
-        RPM_TYPE_HNTCH   = 4,
-        RPM_TYPE_ESC_TELEM  = 5,
+        RPM_TYPE_PIN     = 1,
```

Comment:
Let's keep the enumeration the same as before.  So if "PWM" isn't a valid number anymore, let's just remove "1" as a valid option and start from "2"